public/css/main.css

```diff
@@ -60,3 +60,7 @@ h1 {
 .bn.br--left.br1.ph2:focus {
     outline: none;
 }
+
+.bg-white .orange {
+    color: #a52607;
+}
```

Comment:
Thanks! The banner at the top is still affected as it uses bg-white and orange on the same div. So maybe needs to be done on `.bg-white.orange` as well.